docs/starters.yml

```diff
@@ -3071,3 +3071,19 @@
     - Switch the dark mode according to the system theme
     - Scss
     - Pagination
+- url: https://compassionate-morse-5204bf.netlify.com/
+  repo: https://github.com/deamme/gatsby-starter-prismic-resume
+  description: Gatsby Resume/CV page with Prismic integration
+  tags:
+    - Resume/CV
```

Comment:
```suggestion
```
We currently don't have this tag registered on our starters page. We don't think it's a substantive tag to add to our collection.

Reply:
Alright! Just fixed it